EnvironmentSetup/AWS/Source/deployments/active-web-elements-server-deployment.yaml

```diff
@@ -74,7 +74,8 @@ spec:
             path: /.applicationserver/info
             port: 8080
           initialDelaySeconds: 15
-          periodSeconds: 20
+          periodSeconds: 200
```

Comment:
@delaix This is the suggested number right?

Reply:
No, I don't think we should change this value. We want to check relatively frequently but just not give up so soon. Keep the **periodSeconds** as 20.